lte/gateway/c/core/oai/tasks/amf/amf_smf_send.cpp

```diff
@@ -991,6 +991,11 @@ status_code_e amf_smf_handle_ip_address_response(
         OAILOG_ERROR(LOG_AMF_APP, "Create IPV4V6 Session \n");
       }
     }
+  } else {
+    amf_ue_ngap_id_t ue_id = ue_context->amf_ue_ngap_id;
+    int amf_cause = AMF_CAUSE_PROTOCOL_ERROR;
+    rc = amf_pdu_session_establishment_reject(ue_id, response_p->pdu_session_id,
```

Comment:
This can coded as amf_pdu_session_establishment_reject(e_context->amf_ue_ngap_id, response_p->pdu_session_id, response_p->pti, AMF_CAUSE_PROTOCOL_ERROR);
So that we don't need 2 local variables

Reply:
Addressed.